ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Node/Praos.hs

```diff
@@ -160,7 +162,34 @@ protocolInfoPraosBabbage
     } =
     protocolInfoPraosShelleyBased
       protocolParamsShelleyBased
-      (error "Babbage currently pretending to be Alonzo")
+      (error "Babbage currently pretending to be Alonzo", error "Babbage currently pretending to be Alonzo")
+      protVer
+      maxTxCapacityOverrides
+
+-- | Parameters needed to run Conway
+data ProtocolParamsConway c = ProtocolParamsConway
+  { conwayProtVer :: SL.ProtVer,
+    conwayMaxTxCapacityOverrides :: TxLimits.Overrides (ShelleyBlock (Praos c) (ConwayEra c))
+  }
+
+protocolInfoPraosConway ::
+  forall m c.
+  ( IOLike m,
+    ShelleyCompatible (Praos c) (ConwayEra c),
+    TxLimits (ShelleyBlock (Praos c) (ConwayEra c))
+  ) =>
+  ProtocolParamsShelleyBased (ConwayEra c) ->
+  ProtocolParamsConway c ->
+  ProtocolInfo m (ShelleyBlock (Praos c) (ConwayEra c))
+protocolInfoPraosConway
+  protocolParamsShelleyBased
+  ProtocolParamsConway
+    { conwayProtVer = protVer,
+      conwayMaxTxCapacityOverrides = maxTxCapacityOverrides
+    } =
+    protocolInfoPraosShelleyBased
+      protocolParamsShelleyBased
+      (error "Conway currently pretending to be Alonzo", error "Conway currently pretending to be Alonzo")
```

Comment:
TODO I don't understand this; just copied it from Babbage.

Reply:
I've fixed this now.

I still don't understand why it was that way. I've simply added those stubs as arguments to the function being defined. I'm fairly certain that function is currently dead-code within the greater codebase, so this shouldn't induce any work.

Reply:
I'm Unresolving this, just to double-check it again.

Reply:
I've come to the same conclusion just now as I did in my Sept 7th message earlier in this same comment thread: this function is likely dead, but some downstream 3rd-party _might_ be using it; as a compromise, I've added the stubbed value as an argument.

Reply:
TODO for @dnadales :

- mention this additional parameter in the INTERFACE-changelog.md

- or else make the executive decision to simply remove this function (and also the corresponding new Conway one) instead of adding an argument to it. (Also mention that in the changelog)